components/form-builder/app/responses/DownloadTableDialog.tsx

```diff
@@ -207,6 +233,42 @@ export const DownloadTableDialog = ({
               setStatus={setStatus}
             ></LineItemEntries>
 
```

Comment:
As you can see, not separating these dialogs gets uglier and uglier.. definitely time to refactor soon